modules/launch-templates/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_launch_template" "this" {
+  for_each = local.launch_template_config
+
+  name        = format("%s-%s", each.value.launch_template_prefix, var.eks_cluster_id)
+  description = "Launch Template for Karpenter Nodes"
```

Comment:
Launch Template for Worker Nodes provisioned via Karpenter

Reply:
This is going to be used for other node groups as well. So, updated this to generic description